account_invoice_inter_company/models/account_move.py

```diff
@@ -45,16 +47,34 @@ def create_counterpart_invoices(self):
             # do not consider invoices that have already been auto-generated,
             # nor the invoices that were already validated in the past
             dest_company = src_invoice._find_company_from_invoice_partner()
-            if not dest_company or src_invoice.auto_generated:
-                continue
-            intercompany_user = dest_company.intercompany_invoice_user_id
-            if intercompany_user:
-                src_invoice = src_invoice.with_user(intercompany_user).sudo()
-            else:
-                src_invoice = src_invoice.sudo()
-            src_invoice.with_company(dest_company.id).with_context(
-                skip_check_amount_difference=True
-            )._inter_company_create_invoice(dest_company)
+            if dest_company:
+                if not src_invoice.auto_generated:
+                    intercompany_user = dest_company.intercompany_invoice_user_id
+                    if intercompany_user:
+                        src_invoice = src_invoice.with_user(intercompany_user).sudo()
+                    else:
+                        src_invoice = src_invoice.sudo()
+                    src_invoice.with_company(dest_company.id).with_context(
+                        skip_check_amount_difference=True
+                    )._inter_company_create_invoice(dest_company)
+                if src_invoice.move_type in ["out_invoice", "out_refund"]:
+                    src_invoice._attach_original_pdf_report()
```

Comment:
For doing that, you need sudo permissions always. If you don't enter by first "if" you don't have it, and it fails. If you don't post automatically, when you post manually the customer's invoice renters here but only in second "if"